src/utilities/gql.ts

```diff
@@ -214,6 +231,24 @@ const gql = {
     }
   `,
 
+  CREATE_TAGS: `#graphql
+    mutation CreateTags($tags: [tags_insert_input!]!) {
+      insert_tags(objects: $tags, on_conflict: {
+        constraint: tags_name_key,
+        update_columns: [color]
+      }) {
+        affected_rows
+        returning {
+          color
```

Comment:
Can tag `owner` be changed?

Reply:
Yes - according to @Mythicaeda, only by the owner or admin